src/views/ServiceRequest.js

```diff
@@ -158,8 +161,9 @@ class ServiceRequest extends Component {
       <Checkbox
         label={label}
         name={this.formatLabelToProperty(label)}
+        checked={this.state.form[this.formatLabelToProperty(label)]}
         key={index}
-        onClick={this.handleInputChange}
+        onCheck={this.handleInputChange}
```

Comment:
You can check a checkbox not only by clicking but also by focusing it with tab and pressing space. This works on both cases.

With the previous code it would be checked on screen when you use the keyboard but it would not modify the state.